app.js

```diff
@@ -55,6 +56,14 @@ app.use(function(req, res, next) {
   next()
 })
 
+// set up rate limiter: maximum of five requests per minute
+var limiter = new rateLimit({
+  windowMs: 1*60*1000, // 1 minute
+  max: 120,
+})
+// apply rate limiter to expensive request page(s) - just the one for now
+app.use('/login/dateOfBirth', limiter)
```

Comment:
I was thinking to pass this in as a middleware in the /login route itself. That might solve your issue?

https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/routes/login/login.controller.js#L43

```js
  app.post(
    '/login/dateOfBirth',
    limiter, // pass in here
    checkSchema(dobSchema),
    checkErrors('login/dateOfBirth'),
    postLogin,
    doRedirect,
  )
```

Reply:
The issue has mysteriously gone away. It was also on the checkAnswers page and about the Marital Status row, so it seemed extremely unrelated to begin with.

Are there advantages to changing this to a passed middleware?

Reply:
It looks like cypress works on the first runthrough after a code change, then fails every time after.

Reply:
The advantage would be keeping the middleware with the code itself but this is fine.